make.bat

```diff
@@ -0,0 +1,36 @@
+@ECHO OFF

```

Comment:
Is this just copied from the quickstart?  I'd rather not include it if we haven't tried it.  We can just say that one must build the docs on Linux, not Windows.

Reply:
Yes this is auto generated. Removed.